components/skus/common/skus_sdk.mojom

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+module skus.mojom;
+
+interface SkusSdk {
```

Comment:
Should be `SkusService` instead of `SkusSdk` (since thats the name of the actual service you're writing a mojom interface for)

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/0c492c6a95c3b11eb0f4ab4c316510b78a030d19